oml/utils/audios.py

```diff
@@ -0,0 +1,125 @@
+import base64
+from io import BytesIO
+from typing import Union
+
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+from PIL import Image
+from torch import FloatTensor
+
+from oml.const import BLACK, DEFAULT_SAMPLE_RATE, TColor
+
+
+def default_spec_repr_func(audio: FloatTensor) -> FloatTensor:
+    """
+    Generate a default spectral representation (log-scaled MelSpec) from an audio signal.
+    Used primarily for visualization.
```

Comment:
It's better not to mention how you gonna use the function because it may be changed later or it may push a developer to add a bit more context to the function than it need to know

Reply:
done